lib/index.ts

```diff
@@ -22,53 +25,44 @@ export interface DepTree {
   _isProjSubpkg?: boolean;
 }
 
-export function inspect(root, targetFile) {
+interface CountDict {
+  [k: string]: number;
+}
+
+export async function inspect(root, targetFile) {
 
-  return Promise.all([
+  const result = await Promise.all([
```

Comment:
If you are already there, let's get rid of Promise.all
```suggestion
  const metaDataRes = await getMetaData(root, targetFile);
  const dependenciesRes = await getDependencies(root, targetFile);
```

Reply:
but that would make these operations not parallel. And the idea behind Promise.all was for them to be parallel :)